ethereum/sidecar/server.go

```diff
@@ -264,53 +267,54 @@ func RunServer(
 		server.logger.Info("gRPC server routine stopped")
 	}()
 
-	server.logger.Info(
-		"Waiting for the initial AssetsLocked sync before launching " +
-			"AssetsUnlocked routine",
-	)
+	id, err := server.bridgeContract.ValidatorIDs(accountAddress)
```

Comment:
```suggestion
	bridgeValidatorID, err := server.bridgeContract.ValidatorIDs(accountAddress)
```

Reply:
Done in 6c846c9fa29d8157ddf959efefd3e6122228e6a4.